server.go

```diff
@@ -3981,6 +3981,18 @@ func (s *server) peerConnected(conn net.Conn, connReq *connmgr.ConnReq,
 	// buffers.
 	delete(s.peerErrors, pkStr)
 
+	backup := s.peerBackup.RetrieveBackupForPeer(p.String())
+	if backup != nil {
+		err := p.SendMessage(false, lnwire.NewYourPeerStorageMsg(
+			backup))
+
+		if err != nil {
+			srvrLog.Errorf("unable to send backup to peer "+
+				"%v", err)
+			return
+		}
+	}
```

Comment:
As far as I can tell, we never actually store our peer's blobs so this won't do anything will it?

Reply:
Please point me to any part of this PR that indicates an assumption that the blob is not stored